agent/src/main/java/com/microsoft/applicationinsights/agent/internal/coresync/AgentNotificationsHandler.java

```diff
@@ -62,8 +65,23 @@
      * @param result - The result
      * @param delta - Time it took to do the call
      */
+    @Deprecated
```

Comment:
what is the motivation to mark it as deprecated, why don't just remove it? Could someone call it externally?

Reply:
@lmolkova this is a publically exposed method in an interface and that means any one can directly call it, the likelihood of someone doing it with this method is relatively very very low but still in theory this is a breaking change. So better API releasing norms say to first deprecate and then clean :- )

Reply:
indeed, I thought that the interface is private. deprecation makes perfect sense now